index.html

```diff
@@ -157,7 +157,7 @@ <h3 class="title">Interests</h3>
         <div class="about">
           <h2 class="title2">Curriculum vitae</h2>
           <p>
-            To obtain an internship within my chosen field that will challenge
+            Lmao! To obtain an internship within my chosen field that will challenge
```

Comment:
Add a comment bro , don't touch the content, recruiters do see this resume in my interviews